src/local-settings.js

```diff
@@ -59,16 +60,21 @@ const CONFIGURATION_VARS = [
   [PORTIS_DAPP_ID, process.env.ARAGON_PORTIS_DAPP_ID],
   [FORTMATIC_API_KEY, process.env.ARAGON_FORTMATIC_API_KEY],
 ].reduce(
-  (acc, [option, envValue, envValueCompat]) => ({
+  (acc, [option, ...envValues]) => ({
     ...acc,
     [option]: {
       storageKey: `${option}_KEY`,
-      envValue: envValue || envValueCompat || null,
+      envValue: envValues.reduce((prevVal, envValue) => prevVal || envValue),
```

Comment:
Shouldn't we return `null` here if nothing is found? I believe this may be better than not giving it a default value.

Reply:
Fixed